Use BPM for `cube_sync`

loewenstein opened the pull request:
* monit default for BPM
* no need for ctl script
* no need for cube wrapper script
* add cf-deployment ops file to README.md

loewenstein-sap committed:
Use BPM for `cube_sync`

* monit default for BPM
* no need for ctl script
* no need for cube wrapper script
* add cf-deployment ops file to README.md

loewenstein-sap committed:
Use BPM for `cube_registry`

* monit default for BPM
* no need for ctl script
* no need for cube wrapper script

HeavyWombat commented:
Hello @loewenstein, thank you very much for the PR. @qu1queee and I tested the PR locally and ran into an issue. That can be resolved by changing the order of the operations file for the `deploy`. We would like to ask you to double check what we found and change the README accordingly. The `enable-bpm.yml` needs to be the first operations file, or at least it has to come before a `bosh-lite.yml`.

Also, if you are in the PR, could we ask you to remove `pid_utils` package?

One last thing: Do you think it would be an idea to combine `cube_sync` and `cube_registry` under one bpm config?

loewenstein commented:
@HeavyWombat @qu1queee Sure, will update the PR. Do you prefer an additional commit or `--amend` followed by force push?
I cannot test the `bosh-lite.yml` part easily though as I currently have a non-local CF installation running.

Joining cube_sync and cube_registry in one bpm config would actually make these a single job. I am not sure this is what we want. Will these always run on the same VM? Which one would be the default process? i'd propose to keep them separate unless we find a compelling reason to merge them. cc @julz WDYT?

loewenstein-sap committed:
Remove `pid_utils` package

loewenstein commented:
@HeavyWombat @qu1queee added removal of pid_util package

I had a look at the ops files (bosh-lite and enable-bpm) and couldn't find a potential conflict. Could you provide some details on the issues you had?

loewenstein-sap committed:
bosh-lite.yml removes a job referenced in enable-bpm.yml

Change the order so that first enable-bpm has a change to enable bpm for ssh-proxy before
bosh-lite removes ssh-proxy completely
